Errors while generating javadocs

**Problem description**
Error logs :
```
:clients:streaming:javadoc/var/jenkins_home/workspace/pravega/clients/streaming/src/main/java/io/pravega/stream/StreamSegmentsWithPredecessors.java:37: error: malformed HTML
     * @return Map<Segment, List<Integer>> Segment to Predecessor mapping.
                  ^
/var/jenkins_home/workspace/pravega/clients/streaming/src/main/java/io/pravega/stream/StreamSegmentsWithPredecessors.java:37: error: unknown tag: Integer
     * @return Map<Segment, List<Integer>> Segment to Predecessor mapping.
                                ^
/var/jenkins_home/workspace/pravega/clients/streaming/src/main/java/io/pravega/stream/StreamSegmentsWithPredecessors.java:37: error: bad use of '>'
     * @return Map<Segment, List<Integer>> Segment to Predecessor mapping.
                                         ^
/var/jenkins_home/workspace/pravega/clients/streaming/src/main/java/io/pravega/stream/StreamSegmentsWithPredecessors.java:46: error: malformed HTML
     * @return Map<Segment, Range> segment to range mapping.
                  ^
/var/jenkins_home/workspace/pravega/clients/streaming/src/main/java/io/pravega/stream/StreamSegmentsWithPredecessors.java:46: error: bad use of '>'
     * @return Map<Segment, Range> segment to range mapping.
                                 ^

5 errors
:genJavaDocs UP-TO-DATE
```
**Problem location**
StreamSegmentsWithPredecessors.java

**Suggestions for an improvement**

--- a/clients/streaming/src/main/java/io/pravega/stream/StreamSegmentsWithPredecessors.java
+++ b/clients/streaming/src/main/java/io/pravega/stream/StreamSegmentsWithPredecessors.java
@@ -34,7 +34,7 @@ public class StreamSegmentsWithPredecessors {
     /**
      * Get Segment to Predecessor mapping.
      *
-     * @return Map<Segment, List<Integer>> Segment to Predecessor mapping.
+     * @return A {@link Map} with {@link Segment} as key and {@link List} of {@link Integer} as value.
      */
     public Map<Segment, List<Integer>> getSegmentToPredecessor() {
         return segmentWithPredecessors;
@@ -43,7 +43,7 @@ public class StreamSegmentsWithPredecessors {
     /**
      * Get Segment to Key Range mapping.
      *
-     * @return Map<Segment, Range> segment to range mapping.
+     * @return A {@link Map} with {@link Segment} as key and {@link Range} as value.
      */
     public Map<Segment, Range> getSegmentToRange() {
         return segmentWithKeyRange;

